src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -162,6 +163,14 @@ CylindricalBinaryCompactObject::CylindricalBinaryCompactObject(
   z_cutting_plane_ = cut_spheres_offset_factor_ *
                      ((1.0 - xi) * center_B_[2] + xi * center_A_[2]);
 
+  if (time_dependent_options_.is_time_dependent and
```

Comment:
I think this will never trigger because `is_time_dependent` is always false at this point.